pkg/controllers/runtime_controller.go

```diff
@@ -326,6 +334,15 @@ func (r *RuntimeReconciler) GetDataset(ctx cruntime.ReconcileRequestContext) (*d
 	return &dataset, nil
 }
 
+func (r *RuntimeReconciler) CheckRuntimeSupportDataset(ctx cruntime.ReconcileRequestContext) (bool, string) {
```

Comment:
I suggest renaming `CheckRuntimeSupportDataset` to `CheckIfMountedDatasetIsSupported`.

Reply:
What about `CheckIfReferenceDatasetIsSupported`? I think `mount` may be overused in our case.